frontend/src/components/Board/BoardDescription/BoardDescription.jsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  DescriptionContainer,
+  DescriptionText,
+  PostInfo,
+  Distance,
+  IconsContainer,
+  Icon,
+  BlueLink,
+  BlueText,
+} from './BoardDescription.styled';
+import { FaEye, FaBell, FaPaperPlane } from 'react-icons/fa';
+import { Button } from '../../Button/Button';
+
+const BoardDescription = ({ descriptionText, postsNumber, lastPostDate, distance }) => {
+  const [watch, setWatch] = useState(false);
+  const [notify, setNotify] = useState(false);
+
+  const truncatedText =
+    descriptionText.length > 150 ? descriptionText.substr(0, 149) + '...' : descriptionText;
+
+  return (
+    <DescriptionContainer>
+      <DescriptionText>
+        {truncatedText}
+        {descriptionText.length > 150 ? <BlueLink>&nbsp;Read&nbsp;more</BlueLink> : ''}
```

Comment:
W reakcie do linków, nie używamy elementu 'a' tylko komponentu Link. Chyba, że ma to być odnośnik do innej strony.